tests/suites/helpers.function

```diff
@@ -38,19 +38,25 @@ typedef UINT32 uint32_t;
 #include <unistd.h>
 #endif
 
+/* Type for Hex parameters */
+typedef struct HexParam_tag
+{
+    uint8_t *   x;
+    uint32_t    len;
+} HexParam_t;
```

Comment:
We always use `lowercase_names` for everything except preprocessor macros (and a few variable names in crypto code because math is case-sensitive). Why depart from the convention here?

Reply:
Agreed. ```bin_param_t``` or just ```data_t```.

Reply:
I like `array_t`. That's what it is. Could also do `byte_array_t`.

Reply:
`array_t` makes me wait for you to tell me what it's an array of. I'm happy with any of `buffer_t`, `byte_array_t`, `bin_param_t` or (slightly less) `data_t`.

Reply:
I considered `buffer_t` and `byte_array_t`. Since, it is a custom type I would prefer naming it after it's use, hence `data_t`.